permission_handler/example/lib/main.dart

```diff
@@ -1,127 +1,154 @@
-          actions: <Widget>[
-            IconButton(
-              icon: const Icon(Icons.settings),
-              onPressed: () async {
-                var hasOpened = openAppSettings();
-                debugPrint('App Settings opened: ' + hasOpened.toString());
-              },
-            )
-          ],
+      title: 'Baseflow $pluginName',
+      theme: ThemeData(
+        accentColor: Colors.white60,
+        backgroundColor: const Color.fromRGBO(48, 49, 60, 0.8),
+        buttonTheme: ButtonThemeData(
+          buttonColor: themeMaterialColor.shade500,
+          disabledColor: themeMaterialColor.withRed(200),
+          splashColor: themeMaterialColor.shade50,
+          textTheme: ButtonTextTheme.primary,
+        ),
+        bottomAppBarColor: const Color.fromRGBO(57, 58, 71, 1),
+        hintColor: themeMaterialColor.shade500,
+        primarySwatch: createMaterialColor(const Color.fromRGBO(48, 49, 60, 1)),
+        textTheme: TextTheme(
+          bodyText1: TextStyle(
+            color: Colors.white,
+            fontSize: 16,
+            height: 1.3,
+          ),
+          bodyText2: TextStyle(
+            color: Colors.white,
+            fontSize: 18,
+            height: 1.2,
+          ),
+          button: TextStyle(color: Colors.white),
+          headline1: TextStyle(
+            color: Colors.white,
+            fontSize: 18,
+          ),
         ),
-        body: Center(
-          child: ListView(
-              children: Permission.values
-                  .where((Permission permission) {
-                    if (Platform.isIOS) {
-                      return permission != Permission.unknown &&
-                          permission != Permission.sms &&
-                          //permission != Permission.storage &&
-                          permission != Permission.ignoreBatteryOptimizations &&
-                          permission != Permission.accessMediaLocation;
-                    } else {
-                      return permission != Permission.unknown &&
-                          permission != Permission.mediaLibrary &&
-                          permission != Permission.photos &&
-                          permission != Permission.reminders;
-                    }
-                  })
-                  .map((permission) => PermissionWidget(permission))
-                  .toList()),
+        visualDensity: VisualDensity.adaptivePlatformDensity,
+        inputDecorationTheme: InputDecorationTheme(
+          fillColor: const Color.fromRGBO(37, 37, 37, 1),
+          filled: true,
         ),
       ),
+      home: AppHome(title: 'Baseflow $pluginName example app'),
     );
   }
-}
 
-/// Permission widget which displays a permission and allows users to request
-/// the permissions.
-class PermissionWidget extends StatefulWidget {
-  /// Constructs a [PermissionWidget] for the supplied [Permission].
-  const PermissionWidget(this._permission);
+  /// Creates [MaterialColor] based on [Color]
```

Comment:
Adhere to the Effective Dart style guidelines regarding to Doc comments, specifically rule "PREFER starting library or type comments with noun phrases." (https://dart.dev/guides/language/effective-dart/documentation#prefer-starting-library-or-type-comments-with-noun-phrases)

```suggestion
  /// Creates a [MaterialColor] based on the supplied [Color]
```